base/intset.jl

```diff
@@ -239,3 +245,8 @@ function hash(s::IntSet, h::UInt)
     end
     h
 end
+
+minimum(s::IntSet) = first(s)
+maximum(s::IntSet) = last(s)
+extrema(s::IntSet) = (first(s), last(s))
+issorted(s::IntSet) = true
```

Comment:
Good catch! yes I would say it should be loosened, as long as the method is defined for generic iterators (which makes sense for me).